### PR TITLE
Niassa 2.0.3 fixing tests

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: pipedev-decider-utils/src/test/java/ca/on/oicr/pde/deciders/MergingDeciderBase.java [hunk 1]
```diff
@@ -177,6 +186,32 @@ protected boolean checkFilePassesFilterAfterGrouping(FileAttributes fileAttribut
         protected ReturnValue customizeWorkflowRun(WorkflowRun run) {
             return new ReturnValue();
         }
+        
+        @Override
+        protected List<Map<String,String>> getFileProvenanceReport(Map<FileProvenanceParam, List<String>> params){
```
Comment (morgantaschuk) [inline pipedev-decider-utils/src/test/java/ca/on/oicr/pde/deciders/MergingDeciderBase.java]: This method should probably be removed if the call in BasicDecider was changed back to what it was before. I'm not sure it works.

File: pipedev-decider-utils/src/main/java/ca/on/oicr/pde/deciders/BasicDecider.java [hunk 2]
```diff
@@ -717,14 +717,18 @@ private String createIniFile(String commaSeparatedFilePaths, String commaSeparat
         String iniPath = "";
 
         Map<String, String> iniFileMap = new TreeMap<>();
-        SortedSet<WorkflowParam> wps = metadata.getWorkflowParams(workflowAccession);
-        for (WorkflowParam param : wps) {
-            if (param.getDefaultValue() == null) {
-                Log.debug("Excluding null workflow ini property [" + param.getKey() + "]");
+        Workflow wf = metadata.getWorkflow(Integer.getInteger(workflowAccession));
+        Set<WorkflowAttribute> wps = wf.getWorkflowAttributes();
+        Map<String,String> defaults = wf.getParameterDefaults();
```
Comment (morgantaschuk) [inline pipedev-decider-utils/src/main/java/ca/on/oicr/pde/deciders/BasicDecider.java]: Mike pointed out to me that this is the wrong thing.

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Pull request merged**